dev/sg/main.go

```diff
@@ -354,9 +359,56 @@ func runSetExec(ctx context.Context, args []string) error {
 		cmds = append(cmds, cmd)
 	}
 
+	levelOverrides := logLevelOverrides()
+	for _, cmd := range cmds {
+		enrichWithLogLevels(&cmd, levelOverrides)
+	}
 	return run.Commands(ctx, globalConf.Env, cmds...)
 }
 
+// enrichWithLogLevels will add any logger level overrides to a given command if they have been specified.
+func enrichWithLogLevels(cmd *run.Command, overrides map[string]string) {
+	logLevelVariable := "SRC_LOG_LEVEL"
+
+	if level, ok := overrides[cmd.Name]; ok {
+		out.WriteLine(output.Linef("", output.StyleWarning, "Setting log level: %s for command %s.", level, cmd.Name))
```

Comment:
We need to redo the whole color thing anyway (find a fixed set of colors, use them consistently, etc.) but maybe this?

```suggestion
		out.WriteLine(output.Linef("", output.StylePending, "Setting log level: %s for command %s.", level, cmd.Name))
```

Reply:
Sure thing!